src/syft/core/store/store_memory.py

```diff
@@ -32,7 +32,7 @@ def __init__(self) -> None:
 
     def get_objects_of_type(self, obj_type: type) -> Set[AbstractStorableObject]:
         results = set()
-        for key, obj in self._objects.items():
```

Comment:
NITPICK: Return {obj for obj in self._objects.values() if isinstance(obj, obj_type)} <-- only that it makes the code more concise.